### PR TITLE
Changelog - 14 August 2024

### PR DESCRIPTION
**Pull request by justinegeffen**
Changelog update for cloud release

### CODE REVIEW

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 1]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
 ---
 ## 2024
 
+### 24.2.0.cycle21 - 14 August 2024
+
+- Feature: Add global Nextflow configs in compute environments
+- Feature: Added flexibility for pipeline names in workspaces
+- Feature: Added tags propagation to launch template
+- Feature: Add backend and UI to include managed identities into the configuration when using a manual Azure CE
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:14]:
```suggestion
- Feature: Add managed identities support for manual Azure Batch compute environments
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 2]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
 ---
 ## 2024
 
+### 24.2.0.cycle21 - 14 August 2024
+
+- Feature: Add global Nextflow configs in compute environments
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:11]:
```suggestion
- Feature: Add global Nextflow configuration support in compute environments
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 3]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
 ---
 ## 2024
 
+### 24.2.0.cycle21 - 14 August 2024
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:9]:
```suggestion
### 24.2.0_cycle21 Cloud - 13 August 2024
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 4]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+
+- Feature: Add global Nextflow configs in compute environments
+- Feature: Added flexibility for pipeline names in workspaces
+- Feature: Added tags propagation to launch template
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:13]:
```suggestion
- Feature: Add tag propagation to launch templates
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 5]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+- Feature: Add backend and UI to include managed identities into the configuration when using a manual Azure CE
+- Feature: Implement custom launch container logic
+- Fix: Improve workflow launch L&F
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:16]: What is L&F?

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 6]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+- Fix: Allow changing the workDir when launching a pipeline
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:17]:
```suggestion
- Fix: Allow pipeline work directory to be changed during pipeline launch
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 7]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+- Fix: Properly handle special characters in prompt modal confirmation text regex
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:18]:
```suggestion
- Fix: Handle special characters in prompt modal confirmation text regex
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 8]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+- Bump nextflow 24.04 + codecommit 0.2.1
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:19]: Should this be nf-launcher? nf-launcher:j17-24.04.4 is currently in master

**Review by llewellyn-sl — COMMENTED** [review]
Some suggestions. @swampie please advise on the open questions.

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 9]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+
+- Feature: Add global Nextflow configuration support in compute environments
+- Feature: Added flexibility for pipeline names in workspaces
+- Feature: Add tag propagation to launch templates
+- Feature: Add managed identities support for manual Azure Batch compute environments
+- Feature: Implement custom launch container logic
+- Fix: Improve workflow launch L&F
+- Fix: Allow pipeline work directory to be changed during pipeline launch
+- Fix: Handle special characters in prompt modal confirmation text regex
+- Bump nextflow 24.04 + codecommit 0.2.1
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:19]:
```suggestion
- Bump nf-launcher:j17-24.04.4
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx (hunk 5 above)
Comment (justinegeffen) [reply platform_versioned_docs/version-24.1/cloud/changelog.mdx:16]: Taking it out for now as the source PR is unclear.

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 10]
```diff
@@ -6,9 +6,21 @@ tags: [changelog]
...
+- Feature: Add global Nextflow configuration support in compute environments
+- Feature: Add flexibility for pipeline names in workspaces
+- Feature: Add tag propagation to launch templates
+- Feature: Add managed identities support for manual Azure Batch compute environments
+- Feature: Implement custom launch container logic
+- Fix: Improve workflow launch screen look and feel
+- Fix: Allow pipeline work directory to be changed during pipeline launch
+- Fix: Handle special characters in prompt modal confirmation text regex
+- Bump nf-launcher:j17-24.04.4
+
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:20]:
```suggestion
Bump codecommit 0.2.1

```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 11]
```diff
@@ -6,9 +6,22 @@ tags: [changelog]
...
+- Feature: Add global Nextflow configuration support in compute environments
+- Feature: Add flexibility for pipeline names in workspaces
+- Feature: Add tag propagation to launch templates
+- Feature: Add managed identities support for manual Azure Batch compute environments
+- Feature: Implement custom launch container logic
+- Fix: Improve workflow launch screen look and feel
+- Fix: Allow pipeline work directory to be changed during pipeline launch
+- Fix: Handle special characters in prompt modal confirmation text regex
+- Bump nf-launcher:j17-24.04.4
+Bump codecommit 0.2.1
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:20]:
```suggestion
- Bump codecommit 0.2.1
```

**Review by llewellyn-sl — APPROVED** [review]

**Pull request merged**